addons/hr_expense/static/src/scss/hr_expense.scss

```diff
@@ -20,17 +20,8 @@
 }
 
 .o_dropzone {
-    width: 100%;
-    height: 100%;
-    position: absolute;
     background-color: #AAAA;
     z-index: 2;
     left: 0;
     top: 0;
-    i {
-      justify-content: center;
-      display: flex;
-      align-items: center;
-      height: 100%;
```

Comment:
I think something is missing in your classes :
Render on master :
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/80679690/228551058-cb787b47-27b8-4994-8a62-1bc74e6a8ce8.png">

Render on your branch :
<img width="1791" alt="image" src="https://user-images.githubusercontent.com/80679690/228551101-4e27688c-a18e-4103-871a-7811f136c0d5.png">